infra/cifuzz/fuzz_target_test.py

```diff
@@ -87,5 +88,32 @@ def test_with_invalid_error_string(self):
     self.assertIsNone(self.test_target.get_test_case(' Example crash string.'))
 
 
+class DownloadLatestCorpusUnitTest(unittest.TestCase):
+  """Test parse_fuzzer_output function in the cifuzz module.
+    #NOTE this test relies on arduinojson having the msgpack_fuzzer.
+  """
+
+  def test_download_valid_projects_corpus(self):
+    """Tests that a vaild fuzz target will return a corpus directory."""
+    with tempfile.TemporaryDirectory() as tmp_dir:
+      test_target = fuzz_target.FuzzTarget('testfuzzer', 3, 'test_out')
```

Comment:
Let's make some class variables containing things like arduinojson and reuse them instead.

Reply:
Fixed.